src/Symfony/Component/Notifier/Bridge/Mobyt/MobytOptions.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Mobyt;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @author       Bastien Durand <bdurand-dev@outlook.com>
+ *
+ * @experimental in 5.1
+ */
+final class MobytOptions implements MessageOptionsInterface
+{
+    const MESSAGE_TYPE_HQ = 'N';
```

Comment:
If this means „high quality“

What about MESSAGE_QUALITY_HIGH

?

Reply:
Thanks for tout feedback,
The provider parameter name is message_type that's why I wanted to keep this in the name. 

Here is the description from the doc: 
```
String (“N” for High quality with reception notification, “L” for Medium Quality, “LL” For Low Cost)
```

What do you think of MESSAGE_TYPE_QUALITY_HIGH?

Reply:
Perfect 👌🏻

Reply:
updated the 3 constants.